packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -103,10 +105,14 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     [onEnterKeyDown, onKeyDown, supportNewline]
   );
 
+  let focusText;
+  hasFocus === 'sendBoxTextField' ? (focusText = true) : (focusText = false);
```

Comment:
nit: this can be condensed:
```
const textFieldAutoFocus = hasFocus === 'sendBoxTextField';
```
and given it is a trivial boolean check, I'd move it inline instead of assinging to a new variable for brevity:
```
<TextField autoFocus={props.autoFocus === 'sendBoxTextField'}
```

Reply:
okay sounds good!